elastic-utils-lib/src/cmr/elastic_utils/search/query_execution.clj

```diff
@@ -1,10 +1,14 @@
 (ns cmr.elastic-utils.search.query-execution
+  "Defines a bunch of defmulti functions and their defaults to handle different stats of the
+   query-execution. These will be extended by other files."
   (:require
    [cmr.elastic-utils.search.es-index :as idx]
    [cmr.elastic-utils.search.es-results-to-query-results :as rc]
    [cmr.elastic-utils.search.query-transform :as c2s]
    [cmr.transmit.config :as tc]))
 
+;; *************************************************************************************************
```

Comment:
because this file seamed a bit different then ones I have seen before, it was a list of defmulti calls with default defmethod for each. Obviously this file is meant as a definition of a lot of "interfaces". I found the file slightly unorganized and wanted to highlight the collections of things held here for the future. The way it looks, at least to me, the pairs of calls jump out and the 2 which have a utility function also stand out more.

Reply:
I am always onboard with making code easier to read. Thanks for the explanation!